frontend/pages/MfaPage/MfaPage.tsx

```diff
@@ -84,6 +90,22 @@ const MfaPage = ({ router, params }: IMfaPage) => {
     router.push(paths.LOGIN);
   };
 
+  const onClickFinishLoginButton = () => {
+    setShouldFinishMFA(true);
+  };
+
+  if (!shouldFinishMFA) {
+    return (
+      <AuthenticationFormWrapper>
+        <div className={baseClass}>
+          <Button variant="brand" onClick={onClickFinishLoginButton}>
+            Finish logging in
```

Comment:
What about just:
```suggestion
            Log in
```

It's pretty common to click a log in button and end up redirected to some kind of logged-in dashboard page, so I don't think it'd be weird.